crypto/s390xcpuid.S

```diff
@@ -0,0 +1,428 @@
+#include "s390x_arch.h"
+
+.text
+
```

Comment:
Where has this file come from?

Reply:
Its generated by s390xcpuid.pl, @mattcaswell  probably added it accidentially.

Reply:
Yes I did. I'll remove it.